server/models/services/GithubWorkflowService.js

```diff
@@ -0,0 +1,161 @@
+//import token from '../.github-app-token.json';
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+const path = require('path');
+
+const ONE_MINUTE = 60;
+
+// Assigned by GitHub
+const GITHUB_APP_ID = '395709';
+
+// > your JWT must be signed using the RS256 algorithm.
+//
+// https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app
+const ALGORITHM = 'RS256';
+
+// > Note: [The `workflow_dispatch event] will only trigger a workflow run if
+// > the workflow file is on the default branch.
+//
+// https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+const WORKFLOW_FILE_NAME = 'nvda-chrome.yml';
+const WORKFLOW_REPO = 'bocoup/aria-at-gh-actions-helper';
+
+// Generated from the GitHub.com UI
+let privateKey = null;
+try {
+    privateKey = fs.readFileSync(
+        path.join(
+            __dirname,
+            '../../../config',
+            'aria-at-collection-scheduler.2023-09-27.private-key.pem'
```

Comment:
I can envision this being a little annoying but it might be the best solve; haven't thought through it fully.

Reply:
same - I put it in as a "not sure how best to do this" moment - I'll add a comment that another approach might be better